tests/std/tests/VSO_0157762_feature_test_macros/test.cpp

```diff
@@ -160,7 +160,8 @@ STATIC_ASSERT(__cpp_conditional_explicit == 201806L);
 
 #ifndef __cpp_constexpr
 #error __cpp_constexpr is not defined
-#elif _HAS_CXX20 && defined(__clang__) && __clang_major__ >= 10 // TRANSITION, VSO-951133 and VSO-951142
+#elif _HAS_CXX20 \
+    && (defined(__clang__) && __clang_major__ >= 10 || defined(__EDG__)) // TRANSITION, VSO-951133 and VSO-951142

```

Comment:
Line 267 below adds extra parens around `&&` while this doesn't, which is inconsistent. As the compilers aren't warning, and this is test code, I won't request changes here (ultimately this logic is TRANSITION and short-lived).

Reply:
<3